docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md

```diff
@@ -17,15 +17,35 @@ Search engine optimization (SEO) refers to the methods used to improve your Page
 
 ### HTML Title
 
-The *HTML Title* field defines a Display Page's `<title>` tag. This title is used by search engines to rank your Page and serves as the Page's heading in search engine results. By default, Display Page templates map this field to *Title*, but you can select any other Text field. The recommended length for an HTML title is under 60 characters.
+The *HTML Title* field defines a Display Page's `<title>` tag. This title is used by search engines to rank your Page and serves as the Page's heading in search engine results. By default, Display Page templates map this field to `${title}`, but you can build your title with the mapped fields  that the selector gives you. 
+
+The recommended length for an *HTML title* is under 60 characters.
+
+Default value in HTML title:
+![image](https://user-images.githubusercontent.com/67901/137364376-07ac46f4-fc32-466c-a400-5c498ed6424d.png)
+
+Custom title in HTML title:
+![image](https://user-images.githubusercontent.com/67901/137364659-2f22e395-5a6f-46f1-a847-e424706852b3.png)
+
+For example: `${title} - ${authorName}` produces something like.
+
+```
+<title>New awesome feature - Jane Doe </title>
+```
+
+```note::
+   Take into account that we do not provide translation to the text outside mapped fields, we provide translation for mapped files when is possible.
```

Comment:
Take into account that we do not provide translations to the text outside mapped fields. We provide translations for mapped files when possible.